Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM alpine:3.5
-ENTRYPOINT ["/bin/registrator"]
+# FROM alpine:3.5
+# ENTRYPOINT ["/bin/registrator"]
+#
+# RUN apk --no-cache add ca-certificates
+#
+# COPY ./build/registrator /bin/registrator
+
 
-COPY . /go/src/github.com/gliderlabs/registrator
-RUN apk --no-cache add -t build-deps build-base go git \
-	&& apk --no-cache add ca-certificates \
-	&& cd /go/src/github.com/gliderlabs/registrator \
-	&& export GOPATH=/go \
-  && git config --global http.https://gopkg.in.followRedirects true \
-	&& go get \
-	&& go build -ldflags "-X main.Version=$(cat VERSION)" -o /bin/registrator \
-	&& rm -rf /go \
-	&& apk del --purge build-deps
+FROM golang:1.8.3 as builder
```

Comment:
This is the line CircleCI is choking on.

There is a no "as" provision.

https://docs.docker.com/engine/reference/builder/#from